Added Erik Serrano bio under _members folder (#12)
WayScience/waysciencelab.com: _members/erik-serrano.md

---
name: Erik Serrano

> image:

Reviewer: Please add image!

role: phd
aliases:
  - Erik Serrano
  - Erik Alfonso Serrano
  - Erik A Serrano
  - Erik A. Serrano
  - axiomcura
links:

>   orcid: 0000-0002-0503-9348

Reviewer: Please add your [ORCID](https://orcid.org/)

>   home-page: waysciencelab.com

Reviewer: maybe make it your github profile page?

  email: erik.serrano@cuanschutz.com
  github: axiomcura
  twitter: axiomcura
---

> [ my  bio]

Reviewer: Please write a brief bio - see the site for examples

---
name: Erik Serrano
image: images/team/erik_serrano.png
role: phd
aliases:
    - Erik Serrano
    - Erik Alfonso Serrano
    - Erik A Serrano
    - Erik A. Serrano
    - axiomcura
links:
  orcid: 0000-0003-4188-9080
  home-page: https://github.com/axiomcura
  email: erik.serrano@cuanschutz.com
  github: axiomcura
  twitter: axiomcura
---

Erik Serrano is a Computational Bioscience (CPBS) PhD student at the Univeristy of Colorado, Anschutz Medical Campus (CU-AMC).
He has joined the Way Lab in July 2022, where focuses on developing software applications to analyze cell morphology reads outs.
Before officially joining the Way lab, Erik did conducted his last rotation where he has kick started an application know as [Cytopipe](https://github.com/WayScience/CytoPipe), which possess reproducible analytical pipeline that preprocess high-dimensional cell morphology readouts.

> Erik obtained his bachelors degree in Molecular Biology with a minor in chemistry at the [California State University, Northirdge (CSUN)](https://www.csun.edu/).

Reviewer 2: ```suggestion
Erik obtained his bachelors degree in Molecular Biology with a minor in chemistry at the [California State University, Northridge (CSUN)](https://www.csun.edu/).
```

correct?

> Throughout his undergraduate degree, his research experience mainly pertained to developing computational methods to elucidate biological insights through the usage of protein dynamic data produced by molecular dynamic simulations.

Reviewer 2: ```suggestion
Throughout his undergraduate degree, he developed computational methods to elucidate biological insights from protein dynamic data produced by molecular dynamic simulations.
```

To correct grammar and brevity

> After graduating from CSUN, Erik did his post baccalaureate at [Cedar-Sinai Medical Center](https://www.cedars-sinai.edu/research/departments-institutes/biomedical-sciences.html) where his main job was designing viral based nano-particles through the usage of computational protein modeling.

Reviewer 2: ```suggestion
After graduating from CSUN, Erik did his post baccalaureate at [Cedar-Sinai Medical Center](https://www.cedars-sinai.edu/research/departments-institutes/biomedical-sciences.html) where he designed viral based nano-particles through the usage of computational protein modeling.
```

brevity

In addition, he also developed computational methods using gene expression data to identify potential alternative entry points where the nano-particles can be introduced.

In his free time, Erik likes to learn different romance languages and its origins.

> He is fluent in Spanish and is currently learning Brazilian Portuguese to become his third language.

Reviewer 2: ```suggestion
He is fluent in Spanish and is currently learning Brazilian Portuguese, which will become his third language.
```

Reviewer 2: You're also fluent in python! 😂
